src/schoology-extractor/schoology_extractor/helpers/constants.py

```diff
@@ -17,3 +17,6 @@ class RESOURCE_NAMES:
     ATTENDANCE_EVENTS = "attendance_events"
     DISCUSSION_REPLIES = "discussion_replies"
     DISCUSSIONS = "discussions"
+    SECTION_UPDATE_API = "update"
+    SECTION_UPDATE_TABLE = "section_update"
```

Comment:
I had to define a different name for the table, it had a conflict with the "update" word from sqlite.